setup.py

```diff
@@ -78,69 +78,26 @@ def translate_reqs(packages):
 
 class TestCommand(Command):
 
-    user_options = [
-        ('pytest-args=', 'a', 'Arguments to pass into py.test'),
-        ('fast', 'f', (
-            'Don\'t install dependencies, test in the current environment'
-            )),
-    ]
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
```

Comment:
Are these lines still necessary at all or can they be removed?

Reply:
Command class from setuptools is an abstract class, and these are required fields.